test/TestHelper.cpp

```diff
@@ -526,12 +545,38 @@ bytes importData(json_spirit::mObject const& _o)
 	return data;
 }
 
+std::string compileLLL(std::string const& _code)
+{
+	FILE *fp;
+	char path[1035];
+	FILE *ft = fopen("temp.sol", "w");
+	if (ft == NULL)
+		cerr << "Error creating temp file for lllc";
+	else
+		fputs(_code.c_str(), ft);
+	fclose(ft);
+
+	fp = popen("../../solidity/lllc/lllc temp.sol", "r");
+	if (fp == NULL)
+		cerr << "Failed to run lllc";
+
+	fgets(path, sizeof(path)-1, fp);
```

Comment:
We could check the actual size read and throw an exception if it was `sizeof(path)-1`